pkg/apis/options/legacy_options.go

```diff
@@ -560,6 +561,7 @@ func legacyProviderFlagSet() *pflag.FlagSet {
 	flagSet.String("provider", "google", "OAuth provider")
 	flagSet.String("provider-display-name", "", "Provider display name")
 	flagSet.StringSlice("provider-ca-file", []string{}, "One or more paths to CA certificates that should be used when connecting to the provider.  If not specified, the default Go trust sources are used instead.")
+	flagSet.Bool("provider-ca-file-append", false, "Determines if 'provider-ca-file' files are appended to the default store.  If set to true, files will be appended to the default Go trust stores, otherwise it will replace it.")
```

Comment:
```suggestion
	flagSet.Bool("use-system-trust-store", false, "Determines if 'provider-ca-file' files are and the system trust store are used.  If set to true, your custom CA files and the system trust store are used otherwise only your custom CA files.")
```